src/inference/src/cnn_network_ngraph_impl.cpp

```diff
@@ -259,7 +259,11 @@ void CNNNetworkNGraphImpl::getInputsInfo(InputsDataMap& inputs) const noexcept {
 }
 
 size_t CNNNetworkNGraphImpl::layerCount() const noexcept {
-    return _ngraph_function->get_ops().size();
```

Comment:
May be remove `noexcept`?

Reply:
This is defined as 'noexcept' in base class ICNNNetwork and I just didn't want to change base class and all derived classes for this coverity fix. Do you think it is better to remove 'noexcept' from layerCount and 'getBatchSize'? Can it probbaly affect some 'backward compatibility'?

Reply:
After check with @ilyachur and @ilya-lavrenov it was decided that it would be better remove 'noexcept' keyword, there should not be compatibility issues with this